src/execution/physical_plan/plan_get.cpp

```diff
@@ -36,6 +36,30 @@ unique_ptr<PhysicalOperator> PhysicalPlanGenerator::CreatePlan(LogicalGet &op) {
 			}
 		}
 	}
+	if (!op.zonemap_checks.empty()) {
+		// create the zonemap check filter map
+		zonemap_checks = make_unique<TableFilterSet>();
+		for (auto &tableFilter : op.zonemap_checks) {
+			// find the relative column index from the absolute column index into the table
+			idx_t column_index = INVALID_INDEX;
```

Comment:
This looks like a copy-paste from the code above, could you move it to a function?